### ISSUE TITLE
Time Range button: Keyboard focus is not visible on the selected date in the date picker.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#86661]
### What happened?

Keyboard focus is not visible at all for selected date in date picker dialog. 

https://github.com/grafana/grafana/assets/93735775/a9d8f63c-695e-4b57-a8a6-57cda9e555dc

### What did you expect to happen?

Keyboard focus should be visible properly for selected date in date picker dialog. 

### Did this work before?

No

### How do we reproduce it?

1. Open the URL [Grafana Play Home - Dashboards - Grafana](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1) in edge Dev browser.
2. ‘Grafana Play Home’ page will open.
3. Navigate to ‘Time range’ button & activate it. 
4. Navigate to ‘Date pickers’ present for ‘From’ & ‘To’ fields & activate it. 
5. Navigate to the selected date via tab key. 
6. Observe the issue. 

### Environment (with versions)?

Grafana: v11.1.0-69372 (51da96d94e)
OS: 24H2 (OS Build 26100.1) 
Browser: Edge version 125.0.2506.0 (Official build) dev (64-bit)

### Grafana platform?

None
